test/com/twu/biblioteca/BibliotecaAppTest.java

```diff
@@ -234,3 +234,4 @@ public void cleanUpStreams() {
         System.setErr(null);
     }
 }
+
```

Comment:
Good overall strategy to test the output.
Do you think there might be any issues with changing System.out for the entire JVM?
How would you approach the problem if setOut didn’t exist?